### ISSUE TITLE
Installation breaks with pip -r

### ISSUE THREAD

**Issue by pgcd** [justquick/django-activity-stream#505]
When installing from branch with `pip -r`, the installation fails with this error, due to commit ff0c78d75fbbf50fa95adb2d3ab91d736840221c
```

  Cloning https://github.com/justquick/django-activity-stream.git to /tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6
    ERROR: Command errored out with exit status 1:
     command: /home/vagrant/.virtualenvs/t/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6/setup.py'"'"'; __file__='"'"'/tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-nczys9ho
         cwd: /tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6/setup.py", line 5, in <module>
        from actstream import __version__, __author__
      File "/tmp/pip-install-m96cf74e/django-activity-stream_f33be47027f343d7a8d01206b27189b6/actstream/__init__.py", line 6, in <module>
        import django
    ModuleNotFoundError: No module named 'django'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output
```

**Comment by justquick** [comment]
womp

**Comment by justquick** [comment]
fixed in 1.4.1
